.rubocop.yml

```diff
@@ -70,6 +70,9 @@ Style/HashTransformValues:
 Style/StringLiterals:
   EnforcedStyle: 'single_quotes'
 
+Style/WordArray:
```

Comment:
%w (item1 item2) or ['item1' ,'item2'] 🤷  lets not spend any computational power on that both of them are good understandable concepts.